src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.xml

```diff
@@ -19,7 +19,12 @@
             <argument /><!-- Debug-Flag -->
         </service>
 
+        <service id="annotations.php_file_cache" class="Doctrine\Common\Cache\PhpFileCache" public="false">
+            <argument /><!-- Cache-Directory -->
+        </service>
+
         <service id="annotations.file_cache_reader" class="%annotations.file_cache_reader.class%" public="false">
+            <deprecated>The "%service_id% service is deprecated since 2.8 and will be removed in 3.0."</deprecated>
```

Comment:
yep, but this does not answer my question :) I bet that for bc, we should keep the definition even if it's unlikely that anyone reused this service?